project_recalculate/models/project_project.py

```diff
@@ -64,10 +64,9 @@ def project_recalculate(self):
                     project.date):
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have date end."))
-            if project.calculation_type != 'none':
-                for task in project.tasks:
-                    task.task_recalculate()
-                vals = project._start_end_dates_prepare()
-                if vals:
-                    project.write(vals)
+            for task in project.tasks:
```

Comment:
A ver @antespi , me estoy ahorrando una línea de código, ¿para que voy a poner 
if project.recalculation_type:
   for task in projec.tas:
    (....)

Si antes tengo estas condición:
   if not project.calculation_type:
       raise Warning(_("Cannot recalculate project because your "
                       "project don't have calculation type."))

Si quieres la meto, pero no es para nada necesaria